src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts

```diff
@@ -276,4 +276,39 @@ export class DBiamPersonenkontextRepo {
     public async isRolleAlreadyAssigned(id: RolleID): Promise<boolean> {
         return (await this.findByRolle(id)).length > 0;
     }
+
+    public async getPersonenKontexteWithExceedingBefristung(): Promise<Record<string, Personenkontext<true>[]>> {
```

Comment:
The name is pretty misleading. This should be something like `getPersonenkontexteForPersonsWithExpiredKontexte` or something similar. At least as I understand this code.

Reply:
renamed